src/com/hms/flexyosisoftconnector/OSIsoftConfig.java

```diff
@@ -339,8 +339,8 @@ public static String getOcsToken() {
    *
    * @return the IP Address of the OSIsoft Server
    */
-  public static String getServerIP() {
-    return piServerIP;
+  public static String getServerURL() {

```

Comment:
```suggestion
  public static String getServerUrl() {
  ```

Reply:
As per https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

this applies to all usage of this method as well.